server/routes/notebooks/vizRouter.ts

```diff
@@ -24,7 +24,7 @@ export function registerVizRoute(router: IRouter) {
       request,
       response
     ): Promise<IOpenSearchDashboardsResponse<any | ResponseError>> => {
-      addRequestToMetric('notebooks', 'visualization', 'count');
+      addRequestToMetric('notebooks', 'fetch_visualization', 'count');
```

Comment:
Are there tests that verify this change ?

Reply:
typescript language server verifies this, previously it gave an error but i missed it

```
No overload matches this call. 
Argument of type '"visualization"' is not assignable to parameter of type '"get" | "delete" | "create" | "update" | "add_samples" | "fetch_visualization" | "run_sql_query" | "run_ppl_query"'.
```

Not sure how to add tests for metrics instrumentation calls, let me know if there's a better way, thanks